### ISSUE TITLE
Doesn't Work for SteamInput

### ISSUE THREAD

**Issue by ChrisRevocateur** [jaydenmilne/steamsync#27]
At least for Final Fantasy VII Remake "Path to Game" doesn't work, it will just tell you to start the game through the Epic Launcher.

URL will launch the game, but the Steam Overlay won't work.

The way I've found to launch FFVII Remake that works is to have the target be "C:\Windows\System32\cmd.exe" with launch options set to "/k start com.epicgames.launcher://apps/<insertepicappidhere>?action=launch^&silent=true & exit"

**Comment by flowerdealer** [comment]
URL isn't working for SteamInput for me either, and path to game is giving me issues also with DRM in some games.

**Comment by inshin** [comment]
Have you tried to launch the game through Big Picture? Path mode have to set to Store URL.

**Comment by CANADIANPIG** [comment]
I am having the same issue with Fall Guys. Because the game uses the Epic launcher, the only way to get it to log you in is via the "Epic games store URL" which prevents controller input from working.

**Comment by jaydenmilne** [comment]
Yup, its probably because the Fall Guys process isn’t a child of the Steam process, and doesn’t get SteamInput injected into it.

You probably need something like https://github.com/Alia5/GlosSI (I’ve never used it and can’t really vouch for it, but it looks like that is unsupported right now)

**Comment by CANADIANPIG** [comment]
The workaround I found was to simply set the epic games launcher as the game, and then launch fall guys. This has its own set of issues, but the game is at least playable.

**Comment by dyyd** [comment]
One of the reasons to use "legendary" on windows is the ability to get Epic DRM etc working since it handles launching the game as if it were launched through Epic launcher.
